guides/Getting-Started.md

```diff
@@ -89,10 +89,11 @@ We will use a selection of these commands during this tutorial.
 
 Start by pressing the Tab key to use the autocompletion feature. You will see a collection of names of functions from the Prelude which are available to use.
 
-To see the type of one of these values, first import the appropriate module using the `import` command. `pulp init` configures `.purs-repl` to install `Prelude` automatically, so you won't have to do it yourself.
```

Comment:
@f-f what do you think about having Spago create a `.purs-repl` file including `import Prelude`, like Pulp does, so that the repl "just works"?

Reply:
Having it create this file during spago init, I mean

Reply:
@hdgarrood yeah this is a great idea! I opened https://github.com/spacchetti/spago/issues/555 to track this